_episodes/04-building-containers.md

```diff
@@ -144,6 +147,8 @@ We will automate this in the next section.
 > > ...
 > > ~~~
 > > {: .output}
-> > Notice how we did not need `--fakeroot` for the installation since pip installs user packages.
+> > Notice how we did not need neither `--writable ` nor `--fakeroot` for the installation, but everything worked fine since pip installs user packages, in the user $HOME directory.
+> > And Singularity/Apptainer by default mounts as read+write the user home directory, even if the container is read-only.
+> > This is why a _sandbox_ is great to test and experiment locally but should not be used for containers that will be shared or deployed. Manual changes and local directories are difficult to reproduce and control. Once you are happy with the content you should use definition files, described in the next episode.
```

Comment:
Once you are happy with the content: Add comma

Reply:
added